scissors/build.gradle

```diff
@@ -32,6 +32,7 @@ dependencies {
     provided 'com.squareup.picasso:picasso:[2.4.0, 2.5.2)'
     // Glide
     provided 'com.github.bumptech.glide:glide:[3.5.0, 3.6.1)'
+    provided 'com.nostra13.universalimageloader:universal-image-loader:1.9.4'
```

Comment:
Please update proguard accordingly, also allow to use certain range of versions accordingly e.g  `[1.x.x, 1.9.4)`